java/ql/integration-tests/all-platforms/java/diagnostics/multiple-candidate-builds/diagnostics.expected

```diff
@@ -0,0 +1,28 @@
+{
+  "markdownMessage": "Building using Maven was skipped because there were multiple sibling build directories containing build files: [./maven-project-1,./maven-project-2]. If you want to use one of these, please [TODO](manually supply a build command)",
```

Comment:
```suggestion
  "markdownMessage": "Building using Maven was skipped because there were multiple sibling build directories containing build files: [./maven-project-1,./maven-project-2]. If you want to use one of these, please [manually supply a build command](TODO)",
```